exercises/all-your-base/src/example.c

```diff
@@ -0,0 +1,37 @@
+#include "all_your_base.h"
+#include <math.h>
+
+size_t rebase(int16_t digits[DIGITS_ARRAY_SIZE], int16_t fromBase,
+              int16_t toBase, size_t numDigits)
+{
+   uint16_t denary = 0;
```

Comment:
I had to look this one up. Apparently it's the base-10 equivalent of binary. Neat!